iib/workers/tasks/iib_static_types.py

```diff
@@ -0,0 +1,100 @@
+from operator_manifest.operator import ImageName, OperatorCSV
+
+from typing import TypedDict, Set, List, Dict, Optional
+
+
+class BundleMetadata(TypedDict):
+    """Type class referencing data of bundle used for mypy checking."""
+
+    found_pullspecs: Set[ImageName]
+    operator_csvs: List[OperatorCSV]
+
+
+class IndexImageInfo(TypedDict):
+    """Type class referencing data related to index image used for mypy checking."""
+
+    resolved_from_index: Optional[str]
+    ocp_version: str
+    arches: Set[str]
+    resolved_distribution_scope: str
+
+
+class AllIndexImagesInfo(TypedDict):
+    """Type class referencing group of IndexImageInfo classes used for mypy checking."""
+
+    from_index: IndexImageInfo
+    source_from_index: IndexImageInfo
+    target_index: IndexImageInfo
+
+
+class PrebuildInfo(TypedDict):
+    """Type class referencing data related to preparation of request for building the image."""
+
+    arches: Set[str]
+    binary_image: str
+    binary_image_resolved: str
+    bundle_mapping: Dict[str, List[str]]
+    from_index_resolved: str
+    ocp_version: str
+    distribution_scope: str
+    source_from_index_resolved: str
+    source_ocp_version: str
+    target_index_resolved: str
+    target_ocp_version: str
+
+
+class BundleImage(TypedDict):
+    """Type class referencing data related to bundle image used for mypy checking."""
+
+    packageName: str
+    version: str
+    bundlePath: str
+    csvName: Optional[str]
+
+
+class ResolveImagePullspecs(TypedDict):
```

Comment:
It could be also done without creating the class. 
Instead of class, there could be `resolve_image_pullspecs = TypedDict('resolve_image_pullspecs', {'type':str})`.